src/navs/index.json

```diff
@@ -50,9 +50,9 @@
                                         "url": "/docs/self-host/deploy/digital-ocean"
                                     },
                                     {
-                                        "icon": "github",
-                                        "title": "Source",
-                                        "url": "https://github.com/PostHog/posthog"
+                                        "icon": "azure",
```

Comment:
the icons n00b question was about this - where is the azure icon?

Reply:
The Azure icon is within a `sprites.js` file.

![image](https://user-images.githubusercontent.com/328367/134647512-06f45e4f-52dc-4ca3-9e74-56e8f360ce95.png)